shell/app/modules/msp/monitor/error-insight/pages/errors/error-detail.tsx

```diff
@@ -242,7 +242,7 @@ const ErrorDetail = () => {
                   className={`next-page ${eventIndex === eventIds.length - 1 ? 'edge' : ''}`}
                   onClick={() => changeEvent(1)}
                 >
-                  <span>{i18n.t('msp:next')}</span>
+                  <span>{i18n.t('backup:next')}</span>
```

Comment:
Why use backup space?

Reply:
When I use I18N to execute, the namespace I input is MSP, because the text to be translated here is "next" by default. It uses backup automatically. When I execute I18N, I input MSP, which cannot be overwritten

Reply:
Modified